recipes/mailio/all/conanfile.py

```diff
@@ -88,3 +89,4 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.libs = tools.collect_libs(self)
+        self.cpp_info.requires = ["boost::system", "boost::date_time", "boost::regex", "openssl::openssl"]
```

Comment:
This feature requires conan 1.30.0
Please add `required_conan_version = ">=1.30.0"` before class declaration